.github/ISSUE_TEMPLATE/website-reviewer.yaml

```
@@ -0,0 +1,53 @@
+name: Website Reviewer Nomination Template
+description: Nominate yourself or someone else to become a website reviewer.
+title: 'Website Reviewer Nomination: <username>'
+labels: nomination/website-reviewer
+body:
+  - type: markdown
+    attributes:
+      value: |
+        The Documentation and Community SIG maintains several reviewer roles requiring SIG approval - this nomination form is for serving as a _website reviewer_, responsible for reviewing contributions that add layouts, templates, improve functionality, or change the appearance of [o3de.org](https://www.o3de.org/) in some way. Responsibilities include:
+
+        * Review changes to layouts, partials, and any website development submissions.
+        * Provide guidance on discussions surrounding website development, as needed.
+        * Advocate for quality changes to uphold the website's infrastructure.
+        **Maintaining status:** Serve in the reviewer rotation as required. Serving in the review rotation requires an initial review of the submission,
```

NIt: Add a space here
```suggestion

        **Maintaining status:** Serve in the reviewer rotation as required. Serving in the review rotation requires an initial review of the submission,
```